pyproject.toml

```diff
@@ -127,9 +127,10 @@ exclude = [
     "dbt_common/events/types_pb2.py",
     "venv",
     ".venv",
-    "env*"
+    "env*",
+    ".hatch/*",
 ]
-per-file-ignores = ["*/__init__.py: F401"]
+per-file-ignores = ["*/__init__.py: F401", "*/conftest.py: F401"]
```

Comment:
`pytest` fixtures get registered by importing them into `conftest.py`. They do not need to be used in that file.